cmd/influxd/restore/restore.go

```diff
@@ -309,7 +309,8 @@ func (cmd *Command) unpackTar(tarFile string) error {
 
 // unpackFile will copy the current file from the tar archive to the data dir
 func (cmd *Command) unpackFile(tr *tar.Reader, fileName string) error {
-	fn := filepath.Join(cmd.datadir, fileName)
+	nativeFileName := filepath.FromSlash(fileName)
```

Comment:
If I'm reading the documentation for `filepath.FromSlash` correctly, this will convert `/` characters and leave `\` alone on Windows so this is backwards compatible, right?